packages/base/links-to-many-component.gts

```diff
@@ -110,6 +155,40 @@ class LinksToManyEditor extends GlimmerComponent<Signature> {
         --icon-bg: var(--boxel-dark);
         --icon-border: var(--boxel-dark);
       }
+      .pills {
+        display: flex;
+        flex-wrap: wrap;
+
+        padding: var(--boxel-sp-xs) 0 var(--boxel-sp-xs) var(--boxel-sp-sm);
+        border: 1px solid var(--boxel-form-control-border-color);
+        border-radius: var(--boxel-form-control-border-radius);
+        --boxel-add-button-pill-font: var(--boxel-font-xs);
+        gap: var(--boxel-sp-xs);
+      }
+      .pill-container {
+        position: relative;
+      }
+      .pill {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        box-sizing: border-box;
+        padding: var(--boxel-sp-xxxs) var(--boxel-sp-xs);
+        background-color: var(--boxel-light);
+        border: 1px solid var(--boxel-form-control-border-color);
+        border-radius: var(--boxel-form-control-border-radius);
+        color: var(--boxel-dark);
+        font: 700 var(--boxel-font-xs);
+        letter-spacing: var(--boxel-lsp-xs);
+      }
+      .remove-pill {
```

Comment:
Maybe `remove-item-button` or similar would be a better name?

Reply:
updated [addc943](https://github.com/cardstack/boxel/pull/672/commits/addc943e135e32b1d17e55c97a88c457d63f0a42)